README.md

```diff
@@ -16,10 +16,15 @@
 ## Description
 This terraform module creates an S3 Bucket with the options of adding other s3 bucket configurations.
 
-Examples available [here](https://github.com/boldlink/terraform-aws-s3/tree/main/examples)
+### Reasons to Use this Module over Stand Alone Resources
+- This module is easy to use with elaborate examples
+- Adheres to AWS security best practices by using checkov to enforce compliance
+- Removes the complexity of configuring multiple stand alone resources
+
```

Comment:
Features missing here, including the massive lifecycle changes but also the security settings you have on the code.

Go through the https://github.com/boldlink/terraform-aws-s3/blob/f5632f80cee906ef15fdb3dd25f76de711be8b9b/CHANGELOG.md for a list of the features of this module ;)

Reply:
Added